### DISCUSSION TITLE
Multiple user support

### DISCUSSION THREAD

**Discussion by abirmajum** [lllyasviel/Fooocus#2523] [Q&A] [answered]
I want multiple users to generate images concurrently. Currently, it happens in a queue where the second user has to wait for the first user to finish generating the images. I have multiple GPUs which are capable of running the processes concurrently. Please let me know if there is any way i can achieve this.

**Comment by mashb1t** [comment, marked as answer]
see https://github.com/lllyasviel/Fooocus/issues/292 and https://github.com/lllyasviel/Fooocus/discussions/2292
